### PR TITLE
Deploy the cut-over from accounts.jenkins.io to Azure/K8s

### PR DESCRIPTION
**Pull request by rtyler**

### CODE REVIEW

File: dist/profile/files/bind/jenkins.io.zone [hunk 1]
```diff
@@ -46,14 +46,13 @@ usage       3600    IN    A    52.204.62.78   ; jenkins-usage
 nginx.azure   3600  IN    A       13.68.19.38 ; kubernetes service: namespace:nginx-ingress name:nginx
 plugins.azure 3600  IN    CNAME   nginx.azure    ; used to test plugins.jenkins.io migration
 repo.azure    3600  IN    CNAME   nginx.azure    ; Proxy cache in front of repo.jenkins-ci.org
-beta.accounts 3600  IN    CNAME   nginx.azure    ; Must be deleted when accounts is a CNAME for nginx.azure
+accounts      3600  IN    CNAME   nginx.azure    ; Must be deleted when accounts is a CNAME for nginx.azure
```
Comment (orrc) [inline dist/profile/files/bind/jenkins.io.zone]: Looks like the comment needs to be removed.

**Review by orrc — APPROVED** [review]

File: dist/profile/files/bind/jenkins.io.zone (hunk 1 above)
Comment (olblak) [reply dist/profile/files/bind/jenkins.io.zone]: Nice catch

**Comment by olblak** [comment]
Can this PR also include #842

File: dist/profile/files/bind/jenkins.io.zone (hunk 1 above)
Comment (olblak) [reply dist/profile/files/bind/jenkins.io.zone]: #842 remove this comment

**Review by olblak — APPROVED** [review]

**Pull request merged**